app/lib/actions/katello/content_view/remove.rb

```diff
@@ -74,6 +75,18 @@ def plan(content_view, options)
           end
         end
 
+        def check_version_deletion(versions, cv_envs)
+          versions.each do |version|
+            unless version.environments.empty?
```

Comment:
Do you need this check, given that if `version.environments` is empty, nothing inside the `each` statement will be run?